functions.yml

```diff
@@ -315,8 +329,9 @@ quality_control:
       args:
         - name: genotypes
           doc: The array of genotype structs
```

Comment:
An array of genotype structs with `conditionalQuality` field

Reply:
Done